examples/installation/docker-compose.yml

```diff
@@ -0,0 +1,217 @@
+name: nussknacker
+
+services:
+
+  nginx:
+    image: nginx:1.26.0-alpine
+    restart: unless-stopped
+    ports:
+      - 8080:8080
+    depends_on:
+      grafana:
+        condition: service_healthy
+      designer:
+        condition: service_healthy
+    healthcheck:
+      test: ["CMD", "service", "nginx", "status"]
+      interval: 10s
+      retries: 10
+    volumes:
+      - ./nginx/nginx.conf:/etc/nginx/nginx.conf
+      - ./nginx/additional:/etc/nginx/additional
+
+  ### Nussknacker designer service
+
+  designer:
+    image: touk/nussknacker:1.14.0_scala-2.12
```

Comment:
Why scala 2.12? Let's do it with newer scala, wdyt?

Reply:
I did it intentionally. Until you want to create custom components, you (as a user) don't care about Scala. At the moment Nu with Scala 2.13 is more complicated to install (due to the patching steps of Flink). That's why it's better to use 2.12 in the example. 

AFAIK we will address the problems with Scala 2.13 in the future.